### PR TITLE
Move non-Vivado jobs back to public runners

### PR DESCRIPTION
**Pull request by martijnbastiaan**
Another boring PR :-)

### CODE REVIEW

**Review by cuddlefishie — APPROVED** [review]
This looks good! Just have one question but that's not a blocker :)

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ concurrency:
 
 jobs:
   license-check:
-    runs-on: self-hosted
+    runs-on: ubuntu-22.04
     container:
       image: ubuntu:22.04
```
[the remark this answers is not in the document]
Comment (martijnbastiaan) [reply .github/workflows/ci.yml:22]: No, but! I did do this on purpose: if we need to move back to self hosted runners, it's a matter of `s/ubuntu-22.04/self-hosted/g`.

[the remark this answers is not in the document]
Comment (cuddlefishie) [reply .github/workflows/ci.yml:22]: Yep that makes sense! :+1:

**Review by kleinreact — APPROVED** [review]
Also looks good to me.

**Pull request merged**